requirements.txt

```diff
@@ -4,103 +4,104 @@
 #
 #    pip-compile --output-file=requirements.txt
 #
-ansible==2.8.2            # via teuthology
-apache-libcloud==2.8.0    # via teuthology
+-e file:///home/dependabot/dependabot-updater/tmp/dependabot_20200617-72-1n8af4b  # via -r requirements.in
```

Comment:
see #1515

Reply:
@tchaikov there is another problem the `teuthology` package is replaced with `sanitized-package`, do you know what is that?